code/src/pages/PageNotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Button } from '../components/Button';
+
+import './pagenotfound.css';
+
+export const PageNotFound = () => {
+	return (
+		<div className="page-not-found">
+			<section className="page-not-found__error">
+				<h1>Something went wrong...</h1>
+				<h2>Go back to the movies page and pick another movie.</h2>
+
+				<Button />
+			</section>
+		</div>
+	);
+};
```

Comment:
I've been trying to reach this custom page, but seems like I can only get to the original one?

Reply:
Yayy got it working now!